src/options.js

```diff
@@ -11,8 +13,16 @@ var defaults = {
   bracketSpacing: true
 };
 
+var exampleConfig = Object.assign({}, defaults, {
+  filename: "testFilename",
+  useFlowParser: false,
+  printWidth: 80,
+  originalText: "text"
+});
+
 // Copy options and fill in default values.
 exports.normalize = function(options) {
+  validate(options, { exampleConfig });
```

Comment:
That's true. I think these were supposed to be only printer-specific options and the CLI might have others... but we should probably add the other things like `useFlowParser` here.